sormas-ui/src/main/java/de/symeda/sormas/ui/events/EventsView.java

```diff
@@ -445,8 +450,27 @@ public HorizontalLayout createStatusFilterBar() {
 					}, EntityRelevanceStatus.ARCHIVED.equals(criteria.getRelevanceStatus())));
 
 				bulkOperationsDropdown.setVisible(viewConfiguration.isInEagerMode());
+				bulkOperationsDropdown.setCaption("");
 				actionButtonsLayout.addComponent(bulkOperationsDropdown);
 			}
+
+			if (isDefaultViewType()) {
+				// Contact Count Method Dropdown
+				contactCountMethod = new ComboBox();
+				contactCountMethod.setCaption(I18nProperties.getCaption(Captions.Event_contactCountMethod));
+				contactCountMethod.addItem(0);
```

Comment:
I would prefer to add a dedicated ContactCountMethod enum (can probably be in the UI since it's not needed in the API or backend) with the two values, and therefore to also move the translations to enum.properties. You can then also use this enum for the `setContactCountMethod` method in EventGrid.java instead of passing a boolean.